config/base/numaflow-server/numaflow-server-deployment.yaml

```diff
@@ -38,6 +38,8 @@ spec:
           image: quay.io/numaproj/numaflow:latest
           args:
           - "server"
+          # By default, turn off authentication and authorization.
```

Comment:
I made this change, thinking if we by default enable auth, it could add complexity to first-time numaflow users. @jy4096 also mentioned that we want to disable auth for e2e API testing.

Reply:
cc: @vigith This also replies to your comment above. :)